common/djangoapps/student/tests/factories.py

```diff
@@ -47,6 +48,11 @@ class Meta(object):
     changed_by = None
 
 
+class UserExtendedProfileFactory(DjangoModelFactory):
```

Comment:
This is a core file can't we move this factory somewhere in our code?

Reply:
@qasimgulzar just checked we already have an UserExetendedProfileFactory here `edx-platform/lms/djangoapps/onboarding/tests/factories.py` can we use that?

Reply:
some similar sort of thing is done in `UserFactory` of above mentioned file.